packages/rollup-plugin/src/index.js

```diff
@@ -59,11 +60,20 @@ export default function stylexPlugin({
           useCSSLayers,
         );
 
+        // Process the CSS using lightningcss
+        const { code } = transform({
+          filename: 'stylex.css',
+          code: Buffer.from(collectedCSS),
+        });
+
+        // Convert the Buffer back to a string
+        const processedCSS = code.toString();
```

Comment:
This is a good start, but you must enable the option for vendor prefixing for this to be useful:

https://lightningcss.dev/transpilation.html#browser-targets

Reply:
For the vendor list, would you recommend doing this:
```
let targets = browserslistToTargets(browserslist('>= 0.25%'));
```